### PR TITLE
app-crypt/signify: version bump to 30

### PR DESCRIPTION
**Pull request by epsilon-0**
replaces a lot of older packages in the tree
with their maintained versions
signify
calendar
jot

Package-Manager: Portage-2.3.103, Repoman-2.3.23
Signed-off-by: Aisha Tammy <gentoo@aisha.cc>

### CODE REVIEW

**Comment by gyakovlev** [comment]
I recently dropped `app-shells/loksh` to maintainer-needed. feel free to pick up.

File: sys-apps/outils/outils-0.9.ebuild [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+	EGIT_REPO_URI="https://github.com/leahneukirchen/outils.git"
+else
+	SRC_URI="https://github.com/leahneukirchen/outils/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~sparc ~x86"
+fi
+
+LICENSE="BSD BSD-1 ISC"
+SLOT="0"
+
+#RDEPEND="
```
Comment (epsilon-0) [inline sys-apps/outils/outils-0.9.ebuild:21]: I left this here for now for an idea of how many packages are being blocked if I don't add a prefix to the binaries.

File: sys-apps/outils/outils-0.9.ebuild [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+	!app-misc/calendar
+	!app-misc/jot
+	!net-mail/signify
+	!net-misc/rdate
+	!x11-misc/lndir
+"
+
+src_install() {
+	emake DESTDIR="${D}" PREFIX=/usr install
+	emake DESTDIR="${D}" PREFIX=/usr install-calendars
```
Comment (SoapGentoo) [inline sys-apps/outils/outils-0.9.ebuild:33]: won't work on Prefix then?

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

**Review by SoapGentoo — CHANGES_REQUESTED** [review]
as discussed with @epsilon-0, we'll update the signify in-tree to v30, and push outils into GURU.

File: app-crypt/signify/signify-30.ebuild [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+DEPEND="${RDEPEND}"
+
+PATCHES=( "${FILESDIR}"/${PN}-30-man_compress.patch )
+
+src_configure() {
+	tc-export CC
+}
+
+src_install() {
+	DESTDIR="${D}" PREFIX="/usr" emake install
```
Comment (SoapGentoo) [inline app-crypt/signify/signify-30.ebuild:25]: `emake DESTDIR="${D}" PREFIX="${EPREFIX}"/usr install`

Comment (SoapGentoo) [reply app-crypt/signify/signify-30.ebuild:25]: also, no README or the like for `einstalldocs`

File: app-crypt/signify/signify-30.ebuild [hunk 4]
```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (SoapGentoo) [inline app-crypt/signify/signify-30.ebuild:4]: always leave an empty line after the EAPI definition

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: app-crypt/signify/signify-30.ebuild (hunk 4 above)
Comment (epsilon-0) [reply app-crypt/signify/signify-30.ebuild:4]: done

File: app-crypt/signify/signify-30.ebuild (hunk 3 above)
Comment (epsilon-0) [reply app-crypt/signify/signify-30.ebuild]: done

**Pull request closed without merging**